src/features/toilet/ToiletCard.js

```diff
@@ -78,7 +78,7 @@ function ToiletCard({ toilet, distance, time }) {
 
   function moveToiletDetail(e) {
     e.stopPropagation();
-    navigate(`/toilets/${toiletID}`, {
+    navigate(`/toilets/${_id}`, {
```

Comment:
위와 같은 이유로 아래의 내용을 한번 제안 드립니다. 
```suggestion
    navigate(`/toilets/${toiletId}`, {
```

Reply:
수정완료